packages/fxa-auth-server/test/local/routes/subscriptions/paypal-notifications.js

```diff
@@ -155,4 +156,134 @@ describe('PayPalNotificationHandler', () => {
       sinon.assert.calledOnce(log.error);
     });
   });
+
+  describe('handleMerchPayment', () => {
+    const message = {
+      txn_type: 'merch_pmt',
+      invoice: 'inv-000',
+    };
```

Comment:
It would be nice to have a complete merch_pmt sample message with all the fields they send.